ui/job-view/JobView.jsx

```diff
@@ -26,24 +26,6 @@ const getWindowHeight = function () {
 };
 
 class JobView extends React.Component {
-  static getDerivedStateFromProps(props) {
```

Comment:
It's worth noting that these order changes are the result of a regression in the new version:
yannickcr/eslint-plugin-react#1814


...and once that's fixed, we'll need to move these back.

However I guess that needn't block updating to the newer plugin - we can re-fix again later :-)

Reply:
Oh!  Heh.  I'd thought it was intentional.  How funny.  Yeah, I can move back when an update comes.  No big deal.  Thanks!  :)